en/index.md

```diff
@@ -407,6 +407,51 @@ rather than plain English. Try to keep it simple.
 #21 No double space after a period/full stop
 -->
 
+## Admonitions
+
+Admonitions (also commonly referred to as "admonishments" or sometimes
+"callouts" or "Notifications") are a device used in documentation to draw attention to a
+particular statement or paragraph of text. Typically their use is to highlight:
+
+ - A consequence of performing a particular action
+ - An additional source of information which is useful but not required
+ - A helpful tip that will save effort/time
+ - A reminder of some pre-requisite or restriction
+
+ While this style guide usually concerns itself solely with the intent and
+ structure of content rather than the published form of that content, there is
+ a particular intersect here. The [Vanilla framework][], which is used to
+ publish most of our documentation, only allows for a particular set of
+ pre-defined admonitions (or "Notifications" as they are referred to in that
+ framework): **Information** and **Warning**.
+
+ Note that the style of these admonitions relates solely to the colours and
+ icons used with the notes. It is entirely possible to create an Information
+ admonition and use the label 'Useful tip' or to call a Warning 'Danger of
+ imminent death'.
+
+ Overuse of admonitions diminishes their impact and disrupts the reader's
+ journey, so use them sparingly and try to avoid having several in quick
+ succession.
+
+### Warning
+
+The **Warning** style of admonition is appropriate where certain actions or
+lack of action could have negative consequences - for example the loss of data
+or an interruption to a service.
+
+![An example information notice](https://assets.ubuntu.com/v1/3a5556d1-Screenshot%20from%202024-07-10%2015-13-18.png)
+
+
+### Information
+
+The **Information** admonition should be used for general notices which don't
+form part of the main narrative of the document, such as a useful shortcut, an
+additional source of information or a link to a related tutorial.
+
+![An example warning notice](https://assets.ubuntu.com/v1/c952851c-Screenshot%20from%202024-07-10%2015-11-03.png)
```

Comment:
I'm not sure if these sections are needed -- the particular types of admonition could be deemed self-explanatory and the ones that are available vary between frameworks.

I would suggest moving the example images further up in this section after the admonitions are first described, so that there is a visual aid immediately at hand when the concept is being described.